xbmc/utils/test/TestDatabaseUtils.cpp

```diff
@@ -704,7 +704,7 @@ TEST(TestDatabaseUtils, GetField_MediaTypeEpisode)
                                    DatabaseQueryPartSelect);
   EXPECT_STREQ(refstr.c_str(), varstr.c_str());
 
-  refstr = StringUtils::Format("episode_view.c%02d", VIDEODB_ID_EPISODE_RATING);
+  refstr = StringUtils::Format("episode_view.c%02d", VIDEODB_ID_EPISODE_RATING_ID);
```

Comment:
Now FieldRating use the value directly from the view named rating so refstr should be "episode_view.rating". The same for tv show, movie and musicvideo